cpp/src/hash/xxhash_64.cu

```diff
@@ -18,6 +18,7 @@
 #include <cudf/detail/utilities/algorithm.cuh>
 #include <cudf/hashing/detail/hash_functions.cuh>
```

Comment:
Maybe we can remove this header now as it was needed for `normalize_nans` which has now moved to the header file.

Reply:
Good point. Removed